### DISCUSSION TITLE
Taro Playground App 扫描后没有反应

### DISCUSSION THREAD

**Discussion by Duanruilong** [NervJS/taro#11509] [Q & A]
Taro Playground App 扫描后没有反应
![image](https://user-images.githubusercontent.com/22052028/160063418-d1086bbf-0fce-4a01-8a99-34c558f40f48.png)
![image](https://user-images.githubusercontent.com/22052028/160063492-ac84edf0-61bd-4553-b538-6a49cffd2a73.png)

**Comment by zhiqingchen** [comment]
报错吗？
需要在同一个局域网。
